articles/active-directory/azuread-dev/active-directory-authentication-libraries.md

```diff
@@ -1,15 +1,14 @@
 ---
-title: Azure Active Directory Authentication Libraries | Microsoft Docs
+title: Azure Active Directory Authentication Libraries
```

Comment:
I'm not sure we can change this.  @rwike?

Reply:
#sign-off

Reply:
Note that the site applies "| Microsoft Docs" automatically so it doesn't need to be defined in the metadata. Thanks!